qg/test/testinput/aq_interfaces.yaml

```diff
@@ -30,7 +30,7 @@ state test:
     state variables: *vars
   date: 2010-07-04T01:00:00Z
   norm file: 0.571914995356E-04
-  norm generated state: 1160.96861298991234
+  norm generated state: 1160.9686330528625
```

Comment:
Test now failing on my machine ...
difference between 1160.9686129899121 and 1160.9686330528625 is 2.0062950397914392e-05, i.e. exceeds 1.1609686330528626e-05 (FAIL)